openapi.yml

```diff
@@ -316,8 +316,123 @@ paths:
             application/json:
               schema:
                 type: string
+  /user/{userId}:
+    get:
```

Comment:
How do you feel about:

1. Adding a new `users` tag near the top of the file, near the `groups` tag, and
2. Tagging each of the three new requests with the `users` tag

:question:

Reply:
Added.